tracer/test/Datadog.Trace.Tests/Configuration/ExporterSettingsTests.cs

```diff
@@ -85,23 +211,22 @@ public void Traces_SocketFilesExist_ExplicitUdsConfig_UsesExplicitConfig()
         [Fact]
         public void Traces_SocketFilesExist_ExplicitConfigForWindowsPipeAndUdp_PrioritizesWindowsPipe()
         {
-            var config = Setup(DefaultSocketFilesExist(), "DD_TRACE_PIPE_NAME:somepipe", "DD_APM_RECEIVER_SOCKET:somesocket");
-            Assert.Equal(expected: TracesTransportType.WindowsNamedPipe, actual: config.TracesTransport);
+            var settings = Setup(DefaultTraceSocketFilesExist(), "DD_TRACE_PIPE_NAME:somepipe", "DD_APM_RECEIVER_SOCKET:somesocket");
+            AssertPipeIsConfigured(settings, "somepipe");
         }
 
-        [Fact]
+        [Fact] // fails for now, because Url has no precedence
```

Comment:
I think this was a comment during TDD?

Reply:
Fixed in ddb553a2a9d31b15b72ff19eec7588818af9171e